content/docs/projects/approver-policy.md

```diff
@@ -4,19 +4,19 @@ description: ''
 ---
 
 approver-policy is a cert-manager
-[approver](https://cert-manager.io/docs/concepts/certificaterequest/#approval)
+[approver](../concepts/certificaterequest.md#approval)
```

Comment:
There were a bunch of absolute links which I converted to internal links. Probably because this content was copy - pasted from the README file of the project.